packages/trading-widget/src/core-kit/hooks/trading/use-send-token-debounced-value.ts

```diff
@@ -0,0 +1,22 @@
+import { DEFAULT_DEBOUNCE_TIME, EXTENDED_DEBOUNCE_TIME } from 'core-kit/const'
+import { useSendTokenInput } from 'core-kit/hooks/state'
+import { useDebounce } from 'core-kit/hooks/utils'
+
+export const useSendTokenDebouncedValue = () => {
+  const [sendToken] = useSendTokenInput()
+
+  const debouncedSendTokenValue = useDebounce(
+    sendToken.value,
+    DEFAULT_DEBOUNCE_TIME,
+  )
+  // use extended debounce time to calculate isDebouncing to prevent UI blinking
+  const extendedDebouncedSendTokenValue = useDebounce(
+    sendToken.value,
+    EXTENDED_DEBOUNCE_TIME,
+  )
+
+  return {
+    debouncedSendTokenValue,
+    isDebouncing: sendToken.value !== extendedDebouncedSendTokenValue,
+  }
```

Comment:
what makes 
```
isDebouncing: sendToken.value !== debouncedSendTokenValue,
```
unreliable here?

Reply:
if that's really important. may be we can use smth like: `EXTENDED_DEBOUNCE_TIME * 1.2` to make sure the value will always be bigger than default

Reply:
it helps to prevent slippage blinking during quote loading in `use-deposit-slippage.ts`

Reply:
@artsiomYavorski refactored `useSendTokenDebouncedValue` to be able to receive `extendedDebounceTime` flag